### DISCUSSION TITLE
Request for demo - next-usequerystate with @tanstack/react-table and prisma

### DISCUSSION THREAD

**Discussion by gablabelle** [47ng/nuqs#363] [Q&A] [answered]
Hello there,

It would be nice to have a nice full stack example of using `next-usequerystate` with [@tanstack/react-table](https://tanstack.com/table) and [prisma](https://www.prisma.io/) using the app folder.

This would demonstrate how to reuse parser configurations between client and server code but also how to hook things up properly when all is done server side and where the client simply drives the query params via `next-usequerystate`:

- Filtering
- Sorting
- Pagination
- Can change number of rows per page
- Show/Hide columns
- Default values of all of the above

**Comment by franky47** [comment]
Sounds good, would you like to open a PR?

**Reply by gablabelle** [reply]
I hope I didn't come off as being entitled by requesting this. I simply wanted to provide feedback on what could help the users understand how to properly hook server and client code together. Simply requested the example I wish was there when I started my feature earlier this morning.

Bonne journée.

**Reply by tordans** [reply]
> Simply requested the example I wish was there when I started my feature earlier this morning.

Then let my give a +1 to extract that into an example for here. That would something I could learn from :).

**Reply by franky47** [reply, marked as answer]
Check out `shadcn-table`, it doesn't use Prisma but the level of integration is pretty neat.

https://github.com/sadmann7/shadcn-table
